libs/designsystem/src/lib/components/chart/options/timeseries.ts

```diff
@@ -91,7 +91,7 @@ export const TimeSeriesOptions: Options = {
     {
       type: 'area',
       name: 'USD to EUR',
-      data: this === undefined ? [] : this.data,
+      data: [],
```

Comment:
This doesn't seem like a safe refactoring... 🔥 
See this PR for an alternative solution: https://github.com/kirbydesign/designsystem/pull/1260